code/modules/mob/mob.dm

```diff
@@ -478,6 +479,8 @@ var/next_mob_id = 0
 					mob_eye.observers = list()
 				mob_eye.observers |= src
 				mob_eye.hud_used.show_hud(1,src)
+				var/mob/dead/observer/spook = src
```

Comment:
Ew. Why isn't this just an observer verb, given that only they should ever be able to use it?